docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,141 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.17.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Running Calico Node Containers as a Service
+
+This guide explains how to run Calico as a system process or service. We
+include examples for Systemd, but the commands can be applied to other init
+daemons such as upstart.
+
+## Running the Calico Node Container as a Systemd Service
```

Comment:
I don't think this title should include Systemd.  The whole point is we give an example for systemd, but that this pre-amble is more generic.